spring-cloud-netflix-core/pom.xml

```diff
@@ -36,6 +36,16 @@
 		</dependency>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-security</artifactId>
+			<optional>true</optional>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-test</artifactId>
+			<optional>true</optional>
```

Comment:
Ah ! I miss read your review. I though you were asking why `spring-boot-starter-security` changed to optional:true. Changing `spring-boot-starter-test` to optional is indeed an error. I'll fix it.